app/views/lead_providers/report_schools/start.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="govuk-grid-row">
```

Comment:
I think we're getting copy for this in the next half hour or so

Reply:
I've updated the copy on the page I added.